clients/native/src/commands/mod.rs

```diff
@@ -141,6 +142,10 @@ pub(crate) fn override_config(mut config: Config, args: OverrideConfig) -> Confi
         config.get_base_mut().set_high_default_traffic_volume();
     }
 
+    if args.no_cover {
+        config.get_base_mut().set_no_cover_traffic();
```

Comment:
shouldn't you also disable poisson there given the flag description?

Reply:
`set_no_cover_traffic` disables that too